src/pages/singleDirection.tsx

```diff
@@ -39,6 +39,9 @@ const SingleDirection = ({ operator, lineName, service }: DirectionProps): React
                                 selectNameID="directionJourneyPattern"
                                 journeyPatterns={service.journeyPatterns}
                             />
+                            <span className="govuk-hint hint-text" id="traveline-hint">
+                                This data is taken from the Traveline National Dataset
```

Comment:
Should we also add this to the returnDirection page?

Reply:
Added that line twice - under the dropdown for both Out bound and Inbound - not sure what you think?

Reply:
I think just once under the second dropdown would be enough

Reply:
Now shows just once under the second dropdown.